packages/@lwc/engine-server/src/index.ts

```diff
@@ -26,6 +26,8 @@ export {
     setHooks,
     getComponentDef,
     isComponentConstructor,
+    parseFragment,
+    parseFragment as parseSVGFragment,
```

Comment:
I'm not clear why on the server side SVG can just reuse the other one... I don't remember this.